app/controllers/callbacks/steps_controller.rb

```diff
@@ -50,7 +51,7 @@ def wizard_store
     end
 
     def app_store
-      session[:callbacks] ||= {}
+      session[:callbacks] ||= session[:mailinglist]&.slice("first_name", "last_name", "email", "accepted_policy_id") || {}
```

Comment:
Just thinking - what if a user got interrupted and comes back to the form later when the session expires. Then he won't have those values from the session and the API eventually will fail. Need to put something here to redirect back to the mailinglist form?

Reply:
Yeah that's a good catch; if the various don't exist in the ML session we would want to show them something else, maybe a session expired message or something